sources/operator.1.13.0.psmdb-operator.json

```diff
@@ -53,89 +47,11 @@
             "status": "available",
             "critical": false
           },
-          "4.4.6-8": {
-            "image_path": "percona/percona-server-mongodb:4.4.6-8",
-            "image_hash": "0afd1ae855cd5090d106ba7462b0c1c73e0da7a180b46811b2f3c2149ecb7d4f",
-            "status": "available",
-            "critical": false
-          },
-          "4.4.5-7": {
-            "image_path": "percona/percona-server-mongodb:4.4.5-7",
-            "image_hash": "14b419b686c0ca5cf856e2439bc213b85070ee2fe30832de0f46d1b602cc65ff",
-            "status": "available",
-            "critical": false
-          },
-          "4.4.3-5": {
-            "image_path": "percona/percona-server-mongodb:4.4.3-5",
-            "image_hash": "28c137c87dc0ed3530398ee281b6998a902ca5bc39bf766f42de03b1dfc1fc57",
-            "status": "available",
-            "critical": false
-          },
-          "4.4.2-4": {
-            "image_path": "percona/percona-server-mongodb:4.4.2-4",
-            "image_hash": "991d6049059e5eb1a74981290d829a5fb4ab0554993748fde1e67b2f46f26bf0",
-            "status": "available",
-            "critical": false
-          },
           "4.2.22-22": {
             "image_path": "percona/percona-server-mongodb:4.2.22-22",
             "image_hash": "sha256:da4634df780563e10a547662c58c8ce28fbe5c98e1ac8b42b4f6be87f292e92b",
             "status": "recommended",
             "critical": false
-          },
```

Comment:
Why are we removing all except one 4.2 version?

Reply:
version 1.13.0 works only with 4.2.22-22
cc @hors

Reply:
@tplavcic we have added [js-yaml](https://github.com/percona/percona-docker/commit/c18c1737d1def5d78524f7e740478dd8a8ecf66e) to the latest psmdb 4.2 image. Without this tool operator can't work correctly  with psmdb image (starting from 1.13).

Reply:
I also delete all 4.4 and 5.0 version which don't have this plugin.